tests/mysql_select/mysql_select.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/mysql;
+import ballerinax/mysql.driver as _;
+
+configurable string host = ?;
+configurable string username = ?;
+configurable string password = ?;
+configurable string database_name = ?;
+configurable int port = ?;
+configurable string table_name = ?;
+
+mysql:Client dbClient = check new (host = host, user = username, password = password);
+
+service /db on new http:Listener(9092) {
+    resource function post .() returns string|error {
+        stream<record {}, error?> resultStream =
+                dbClient->query("SELECT COUNT(*) AS total FROM " + database_name + "." + table_name);
```

Comment:
Better to use parameterized queries and typed return stream as these are the most common use case

Reply:
Do we need the table name configurable? We expect the table name to be explicitly defined in the query during production.

Reply:
We are creating DB and table during the MySQL deployment. So, I thought that It better to get the table name from config.

Reply:
It's good maintainability wise but I was thinking since we can point these when users look for performance figures, it's better to align with common usecases.

Reply:
yes, shall we write with the parameterized query? We can use these samples as references

Reply:
Fixed in 0f85e97